core/src/main/scala/akka/persistence/cassandra/EventsByTagMigration.scala

```diff
@@ -105,23 +108,25 @@ class EventsByTagMigration(
     with CassandraTagRecovery {
 
   private[akka] val log = Logging.getLogger(system, getClass)
-  private val queries = PersistenceQuery(system).readJournalFor[CassandraReadJournal](readJournalNamespace)
+  private lazy val queries = PersistenceQuery(system).readJournalFor[CassandraReadJournal](readJournalNamespace)
```

Comment:
Don't want the journal to connect eagerly and do its initialisation and get the metadata as this tests changes the tables